platooning_strategic_IHP/src/platoon_strategic_ihp.cpp

```diff
@@ -688,6 +857,8 @@ namespace platoon_strategic_ihp
                 request.location = pose_to_ecef(pose_msg_);
                 request.plan_type.type = cav_msgs::PlanType::JOIN_PLATOON_AT_REAR;
                 request.strategy = MOBILITY_STRATEGY;
+                // UCLA: Desired joining index for cut-in join, indicate the index of gap-leading vehicle. -1 indicate cut-in from front.
+                request.join_index = -1; // Note: not used by same-lane functions.
```

Comment:
I don't understand why this is here, if it is not used for same-lane join, which is what this code block is doing.  In any case, -1 indicates front join and this is the rear join logic.